datasets/ph/most_wanted/crawler.py

```diff
@@ -0,0 +1,36 @@
+import csv
+from typing import Dict
+import zavod.helpers as h
+from zavod import Context
+
+
+def crawl_row(context: Context, row: Dict[str, str]):
+    data = dict(row)
+    full_name = data.pop("name", None)
+    offense = data.pop("offense", None)
+    case_number = data.pop("case number", None)
+    source = data.pop("source", None)
+    reward = data.pop("reward", None)
```

Comment:
I think it's best to drop these None defaults - let's get alerted if any of these columns disappear